src/Lykke.Job.OperationsCache.AzureRepositories/CashOperations/TransferEventsRepository.cs

```diff
@@ -39,6 +40,22 @@ public TransactionStates State
         }
         public string ClientId { get; set; }
 
+        public double FeeSize { get; set; }
+        public FeeType FeeType { get; set; }
+        public string FeeTypeText
```

Comment:
Is this correct that both two properties are writable? I think one should be readonly as it derives from the second.